pdns/common_startup.cc

```diff
@@ -154,7 +154,9 @@ void declareArguments()
   ::arg().setSwitch("do-ipv6-additional-processing", "Do AAAA additional processing")="yes";
   ::arg().setSwitch("query-logging","Hint backends that queries should be logged")="no";
 
+  ::arg().set("carbon-namespace", "If set overwrites the first part of the carbon string)=");
```

Comment:
Why not set this to `pdns` here? This saves the 

```cpp
if(namespace_name.empty()) {..}
```

check in auth-carbon later on.